src/ec/packages/ec-component-icon/icon.html.twig

```diff
@@ -0,0 +1,51 @@
+{#
+  Parameters:
+    - "icon" (associative array) (default: predefined structure): format: 
+        {
+          path: '/static/media/icons.cbfd6efe.svg',
+          type: '',
+          name: '',
+          size: 'm' (available options: 'xs','s','m','l','xl','2xl','fluid'),
+          transform: 'rotate-0' (available options: 'rotate-0', 'rotate-90', 'rotate-180', 'rotate-270', 'flip-horizontal', 'flip-vertical'),
+          color: 'default' (available options: 'default', 'inverted', 'primary')
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+{% set _icon={
+  path: '/static/media/icons.cbfd6efe.svg',
```

Comment:
See my comment below: https://github.com/ec-europa/ecl-twig/pull/12#issuecomment-457176708

> I am still wondering what you wrote about PATH - from the Twig template side we always have to provide such PATH as a variable, but from the JavaScript side it can be any form - a global variable or a variable in the component.